tools/demo/README.md

```diff
@@ -1,6 +1,13 @@
 How to run
 ----------
 
+0. Spin up the required dependency containers:
```

Comment:
I didn't even know you could start at 0?  It's usually wise in markdown files to number all the items in your list as `1.` from the off.  That way you don't end up in situations where adding items in the middle of the list causes you to have to refactor the whole list, or do what you did here.

Reply:
Well I did what they did when naming the zeroth law of thermodynamics 😆 

I was going to refactor the list later and forgot. I'll refactor it 😄